Platforms/QemuQ35Pkg/QemuQ35Pkg.dsc

```diff
@@ -909,9 +910,36 @@ PlatformSmmProtectionsTestLib|UefiTestingPkg/Library/PlatformSmmProtectionsTestL
   gEfiMdeModulePkgTokenSpaceGuid.PcdPlatformRecoverySupport|FALSE
   gPcBdsPkgTokenSpaceGuid.PcdLowResolutionInternalShell|FALSE
   # The GUID of SetupDataPkg/ConfApp/ConfApp.inf: E3624086-4FCD-446E-9D07-B6B913792071
+
+!if $(GUI_FRONT_PAGE) == TRUE
+# Note:
+#  The PcdBootManagerMenuFile Pcd does two things:
+#    1.  It sets the default front page (the one entered exiting the shell, or if all boot
+#        boot options fail), and:
+#    2.  If this application is the first boot option being loaded, Bds does not call ReadyToBoot
+#        to lock the private settings variables.
+#
+#  To use this feature:
+#
+#    1. build Q35Pkg with BLD_*_GUI_FRONT_PAGE=TRUE
+#    2. Run Q35

```

Comment:
Since this is open source, I think it's better to be a bit more precise to reduce potential confusion.

```suggestion
#    1. Build QemuQ35Pkg with BLD_*_GUI_FRONT_PAGE=TRUE
#    2. Run QEMU

```